src/queues.c

```diff
@@ -355,9 +363,21 @@ void queues_update(bool fullscreen)
                 notification *n = iter->data;
                 GList *nextiter = iter->next;
 
-                if (displayed_limit > 0 && displayed->length >= displayed_limit) {
-                        /* the list is full */
-                        break;
+                if (displayed_limit > 0) {
```

Comment:
Note to myself: What to do, if `displayed_limit = 1` (the case when the user only wants to show a single notification at all times)?